src/Proto.Cluster/Partition/PartitionIdentityActor.cs

```diff
@@ -63,12 +65,28 @@ private Task OnStarted(IContext context)
 
         private async Task OnClusterTopology(ClusterTopology msg, IContext context)
         {
-            await _cluster.MemberList.TopologyConsensus();
-            if (_topologyHash == msg.TopologyHash) return;
+            for (int i = 0; i < 3; i++)
+            {
+                try
+                {
+                    await OnClusterTopologyInner(msg, context);
```

Comment:
Hackery, topology handover needs to be improved, large clusters tend to storm when that happens, 50*50 nodes all shooting handover at eachother